src/components/MentionSuggestions.tsx

```diff
@@ -98,39 +89,37 @@ function MentionSuggestions(props) {
                     style={[styles.mentionSuggestionsText, styles.flex1]}
                     numberOfLines={1}
                 >
-                    {_.map(
-                        styledHandle,
-                        ({text, isColored}, i) =>
-                            text !== '' && (
-                                <Text
-                                    key={`${text}${i}`}
-                                    style={[StyleUtils.getColoredBackgroundStyle(isColored), styles.mentionSuggestionsHandle, {...(isColored && {color: styles.text})}]}
-                                >
-                                    {text}
-                                </Text>
-                            ),
-                    )}
+                    {styledHandle &&
+                        styledHandle.map(
+                            ({text, isColored}) =>
+                                text !== '' && (
+                                    <Text
+                                        key={`${text}${isColored}`}
+                                        style={[StyleUtils.getColoredBackgroundStyle(isColored), styles.mentionSuggestionsHandle, {...(isColored && {color: themeColors.text})}]}
```

Comment:
Why did you change `styles.text` to `themeColors.text`?

Reply:
we don't have such property anymore, changed to the property with same value

Reply:
removed, since it never exists